src/genworld_gui.cpp

```diff
@@ -183,6 +183,16 @@ static const NWidgetPart _nested_generate_landscape_widgets[] = {
 				NWidget(NWID_SPACER), SetFill(1, 1),
 			EndContainer(),
 		EndContainer(),
+		NWidget(NWID_SPACER), SetMinimalSize(0, 6), SetFill(1, 1),
+		/* NewGRF and AI/GS Settings */
+		NWidget(NWID_HORIZONTAL, NC_EQUALSIZE), SetPIP(10, 0, 10),
+			NWidget(WWT_PUSHTXTBTN, COLOUR_ORANGE, WID_GL_NEWGRF_BUTTON), SetMinimalSize(0, 24), SetDataTip(STR_MAPGEN_NEWGRF_SETTINGS, STR_NULL), SetFill(1, 0),
```

Comment:
I could go either way on this. Note that the Generate button is already bigger because it's important. 😉

Reply:
I personally did this because I felt that if I put a single height row below the water edge selections it would disturb the flow of the UI too much and be harder to see, it wasn't just because it was important, though @2TallTyler's point is valid. Also, it discerns these 3 from the rest of the settings, which are land generation based; These 3 are outside of that scope.

Reply:
Maybe a panel might also work out here. GUI design is hard :(